examples/findfiles_up.cf

```diff
@@ -0,0 +1,39 @@
+#+begin_src prep
+#@ ```
+#@ mkdir -p /tmp/repo/.git/
+#@ touch /tmp/repo/.git/config
+#@ mkdir -p /tmp/repo/submodule/.git/
+#@ touch /tmp/repo/submodule/.git/config
+#@ mkdir -p /tmp/repo/submodule/some/place/deep/within/my/repo/
+#@ ```
+#+end_src
+###############################################################################
+#+begin_src cfengine3
+body common control
+{
```

Comment:
Remove body common control, rename `bundle agent example` to `bundle agent __main__`